tests/unit/resources/test_datastores.py

```diff
@@ -173,6 +173,20 @@ def test_resize(self, mock_get, mock_post):
         self.assertEqual(datastore.data, resource_data[0])
         mock_post.assert_called_once_with('/datastores/12345/resize', {'size': 2048}, custom_headers=None)
 
+    @mock.patch.object(Connection, "post")
+    @mock.patch.object(Connection, "get")
+    def test_set_policy(self, mock_get, mock_post):
+        resource_data = [{'id': '12345', 'name': 'ds1', 'policy_id': '6789'}]
+        mock_get.return_value = {datastores.DATA_FIELD: resource_data}
+        mock_post.return_value = None, [{'object_id': '12345'}]
+        datastore_data = {'name': 'name1', 'id': '12345', 'policy_id': '4567'}
```

Comment:
it doesn't affect the test, but it is confusing that the name of the datastore changes between object creation, and set policy.

i.e.  line 179 the name is ds1, but line 182 the name is changed to name1

Reply:
Updated the name to ds1